modules/seqexec/web/client/src/main/scala/seqexec/web/client/model/SequencesOnDisplay.scala

```diff
@@ -395,17 +395,13 @@ object SequencesOnDisplay {
   def loadingL(id: Observation.Id): Traversal[SequencesOnDisplay, Boolean] =
     SequencesOnDisplay.previewTabById(id) ^|-> PreviewSequenceTab.isLoading
 
-  def tabG(id: Observation.Id): Getter[SequencesOnDisplay, SeqexecTabActive] =
+  def tabG(id: Observation.Id): Getter[SequencesOnDisplay, Option[SeqexecTabActive]] =
```

Comment:
This is the core change. I cheated a bit by defining an `EmptySeqexecTab` but it's probably better to pass the `Option` up the stack